ecommerce/frontend/src/components/pages/LoginPage.js

```diff
@@ -1,3 +1,107 @@
-export const LoginPage = () => {
-    return 'login'
-}
+import './LoginPage.less'
+import { Form, Input, Button, Row, Col } from 'antd'
+import { Link } from 'react-router-dom'
+import Column from 'antd/lib/table/Column'
+
+export const LoginPage = ({ onSubmit = () => { } }) => {
+    const [form] = Form.useForm()
+
+    const onFinish = values => {
+        console.log('Received values of form: ', values)
+        onSubmit(values)
+    }
+
+    const onFinishFailed = errors => {
+        console.log('Finish failed: ', errors)
+    }
+
+    const onLoginAsVendor = () => {
+        const loginInfo = {
+            "email": form.getFieldValue("email"),
+            "password": form.getFieldValue("password")
+        }
+        console.log(loginInfo)
+    }
+
+    // Constants for layout
+    const layout = {
+        labelCol: {
+            span: 8,
+        },
+        wrapperCol: {
+            span: 16,
+        }
+    };
+    const tailLayout = {
+        wrapperCol: {
+            offset: 8,
+            span: 16,
+        }
+    };
+
+    // TO DO ln: 96
+    // Implement Google Login
+    return (
+        <div className="loginForm">
+            <Form
+                {...layout}
+                layout="horizontal"
+                form={form}
+                name="loginForm"
+                onFinish={onFinish}
+                onFinishFailed={onFinishFailed}
+                scrollToFirstError>
+
+                {/* Email input */}
+                <Form.Item
+                    name="email"
+                    label="E-mail"
+                    rules={[
+                        {
+                            type: 'email',
+                            message: 'The input is not valid E-mail!',
+                        },
+                        {
+                            required: true,
+                            message: 'Please input your E-mail!',
+                        },
+                    ]}>
+                    <Input />
+                </Form.Item>
+
+                {/* Password input */}
+                <Form.Item
+                    name="password"
+                    label="Password"
+                    rules={[
+                        {
+                            required: true,
+                            message: 'Please input your password!',
+                        },
+                    ]}>
+                    <Input.Password />
+                </Form.Item>
+
+                <div className="loginButtons">
+                    <Form.Item {...tailLayout}>
+                        <Button className="loginButton" type="primary" htmlType="submit">
+                            Login as User
+                        </Button>
+                        <Button className="loginButton" type="primary" htmlType="button" onClick={onLoginAsVendor}>
+                            Login as Vendor
```

Comment:
What is the difference between Login button html types. Doesn't both send HTTP requests to the server?

I looked at the MDN page to understand more. Submit type sends request and button has no default action, so a function is required. 

One radio element from antd to select login user type and one button to submit form values might be easier. (Ozdeniz had used Radio component, you can copy-paste from his commit)

Reply:
@abdullahyldz I believe the "default action" associated with htmlType=submit is blocked by antd so it only serves a semantic purpose like this is the main submit function falan. But I think both here need to be regular buttons with their own callback to differentiate between vendor and customer.

Also please change User to Customer.